INSTALL

```diff
@@ -1,54 +1,51 @@
-qBittorrent - A BitTorrent client in C++ / Qt4
+qBittorrent - A BitTorrent client in C++ / Qt
 ------------------------------------------
 
-1) Compile and install qBittorrent with Qt4 Graphical Interface
+1) Compile and install qBittorrent with Qt graphical interface
 
   $ ./configure
   $ make && make install
   $ qbittorrent
 
-  will install and execute qBittorrent hopefully without any problems.
+  Will install and execute qBittorrent.
 
   Dependencies:
-    - Qt >= 4.6.0 (libqtgui, libqtcore, libqtnetwork, libqtxml, libqtdbus/optional)
+    - Qt >= 5.5.1
 
-    - pkg-config executable
+    - pkg-config
 
-    - libtorrent-rasterbar by Arvid Norberg (>= 1.0.6)
-        -> http://www.libtorrent.net
-        Be careful: another library (the one used by rTorrent) uses a similar name.
+    - libtorrent-rasterbar >= 1.0.6 (by Arvid Norberg)
+        * https://www.libtorrent.org/
+        * Be careful another library (the one used by rTorrent) uses a similar name
 
-    - libboost >= 1.35.x (libboost-system)
+    - Boost >= 1.35
+        * libboost-system
```

Comment:
You did not list even Qt modules (qtbase, qttools) but do mention a single library name? Besides, this is a libtorrent dependency and you don't list all of them.